dart/lib/src/sentry_options.dart

```diff
@@ -184,6 +184,14 @@ class SentryOptions {
   /// sent. Events are picked randomly. Default is null (disabled)
   double? sampleRate;
 
+  /// The ignoreErrors tells the SDK which errors should be not sent to the sentry server.
+  /// If an null or an empty list is used, the SDK will send all transactions.
+  List<String>? ignoreErrors;
```

Comment:
```suggestion
  List<String> ignoreErrors = [];
```

we can keep this by default as empty so we save ourselves from null checks and !

same for `ignoreTransactions`